src/DotNetty.Codecs.Redis/RedisCodecUtil.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Codecs.Redis
+{
+    using System;
+    using System.Globalization;
+    using System.Text;
+    using DotNetty.Buffers;
+    using DotNetty.Codecs.Redis.Messages;
+
+    static class RedisCodecUtil
+    {
+        internal const char RedisSimpleString = '+';
+        internal const char RedisError        = '-';
+        internal const char RedisInteger      = ':';
+        internal const char RedisBulkString   = '$';
+        internal const char RedisArray        = '*';
+
+        internal static RedisMessageType ParseMessageType(byte byteCode)
+        {
+            switch ((char)byteCode)
+            {
+                case RedisSimpleString:
+                    return RedisMessageType.SimpleString;
+                case RedisError:
+                    return RedisMessageType.Error;
+                case RedisInteger:
+                    return RedisMessageType.Integer;
+                case RedisBulkString:
+                    return RedisMessageType.BulkString;
+                case RedisArray:
+                    return RedisMessageType.ArrayHeader;
+                default:
+                    throw new RedisCodecException($"Unknown RedisMessageType code:{byteCode}");
+            }
+        }
+
+        internal static byte[] LongToAsciiBytes(long value) => 
```

Comment:
pls add a comment: `// todo: can optimize once ByteBufferUtil.WriteAscii is implemented`